src/test/java/foodwhere/storage/JsonAdaptedStallTest.java

```diff
@@ -13,84 +13,66 @@
 import foodwhere.commons.exceptions.IllegalValueException;
 import foodwhere.model.stall.Address;
 import foodwhere.model.stall.Name;
-import foodwhere.model.stall.Phone;
 
 public class JsonAdaptedStallTest {
     private static final String INVALID_NAME = "R@chel";
     private static final String INVALID_PHONE = "+651234";
     private static final String INVALID_ADDRESS = " ";
-    private static final String INVALID_DETAIL = "#friend";
+    private static final String INVALID_EMAIL = "example.com";
```

Comment:
Perhaps we can remove this field as it is not used anywhere?

Reply:
Removing this and `INVALID_PHONE`.